nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -1965,6 +1919,37 @@ def _greedy_decode_blank_as_pad(
                                 hypotheses[kidx].score += float(v[kidx])
 
                         symbols_added += 1
+                else:
+                    for i in range(len(big_blank_masks) + 1):
+                        # The task here is find the shortest blank duration of all batches.
+                        # so we start from the shortest blank duration and go up,
+                        # and stop once we found the duration whose corresponding mask isn't all True.
+                        if i == len(big_blank_masks) or not big_blank_masks[i].all():
+                            big_blank_duration = self.big_blank_durations[i - 1] if i > 0 else 1
+                            break
```

Comment:
This `break` was inside the while loop, but now it's inside the upper `for`. Is this intended? Looks like some kind of logic change is required in addition to just moving this outside of the loop?

Reply:
Nope, the break was inside for. I'm pretty sure I'm not crazy😁

Reply:
Ah, right, sorry about that! :)